android/library/src/main/java/com/liferay/mobile/screens/rating/RatingScreenlet.java

```diff
@@ -117,12 +136,19 @@ protected void onUserAction(String userActionName, Interactor interactor, Object
 		}
 	}
 
+	/**
+	 * Checks if there is a session created and then calls {@link #load()} method.
+	 */
 	protected void autoLoad() {
-		if (SessionContext.isLoggedIn()) {
+		if (SessionContext.isLoggedIn() && (entryId != 0 || (className != null && classPK != 0))) {
```

Comment:
We should try to add/fix bugs in other branches and only comment code in this one